pyproject.toml

```diff
@@ -0,0 +1,13 @@
+[tool.pytest.ini_options]
+pythonpath = [
+  "."
+]
+markers = [
+    "slow: marks tests as slow (deselect with '-m \"not slow\"')",
+    "serial",
+    "api: marks tests as API tests (deselect with '-m \"not api\"')",
+    "unit: marks tests as unit tests (deselect with '-m \"not unit\"')",
+    "integration: marks tests as integration tests (deselect with '-m \"not integration\"')",
+    "functional: marks tests as functional tests (deselect with '-m \"not functional\"')",
+    "data: marks tests as data-related tests (deselect with '-m \"not data\"')"
+]
```

Comment:
just added some potentially useful markers (we can add/remove as needed) so we can mark tests with `@pytest.mark.<marker>`

Reply:
Did not know about this .toml file. Nice to learn new stuff. Tell me a bit more about how can we use it

Reply:
right now it's only used for `pytest`, but if we want to package the entire thing as a python module then you can use the `.toml` file for that too. [https://pip.pypa.io/en/stable/reference/build-system/pyproject-toml/](Mainly used by pip, but others including `pytest` can utilize it).